pkg/services/playlist/playlist.go

```diff
@@ -7,8 +7,8 @@ import (
 type Service interface {
 	Create(context.Context, *CreatePlaylistCommand) (*Playlist, error)
 	Update(context.Context, *UpdatePlaylistCommand) (*PlaylistDTO, error)
-	Get(context.Context, *GetPlaylistByUidQuery) (*Playlist, error)
-	GetItems(context.Context, *GetPlaylistItemsByUidQuery) ([]PlaylistItem, error)
+	GetSummary(context.Context, *GetPlaylistByUidQuery) (*Playlist, error)
+	Get(context.Context, *GetPlaylistByUidQuery) (*PlaylistDTO, error)
```

Comment:
This is a strange API, naming wise Get returns a DTO and GetSummary returns a non DTO , GetSummary (the function name) tells me almost nothing. Summary of what? Its it Get without child references?